js/ui/map.js

```diff
@@ -100,7 +100,7 @@ const defaultOptions = {
  *   Keep in mind that these classes are used for controlling a style layer's paint properties, so are *not* reflected
  *   in an HTML element's `class` attribute. To learn more about Mapbox style classes, read about
  *   [Layers](https://www.mapbox.com/mapbox-gl-style-spec/#layers) in the style specification.
- * @param {boolean} [options.attributionControl=true] If `true`, an [AttributionControl](#AttributionControl) will be added to the map.
```

Comment:
Did you really mean to replace `attributionControl` with `logoPosition`, or did you mean to replace `logoControl` with `logoPosition`?

Given further down you've dropped out `options.logoControl` with `options.logoPosition` but retained `options.attributionControl`.